# Fix C includes

diff -N -r -U 3 mysql-connector-c++-1.0.5-orig/driver/mysql_art_resultset.cpp mysql-connector-c++-1.0.5/driver/mysql_art_resultset.cpp
--- mysql-connector-c++-1.0.5-orig/driver/mysql_art_resultset.cpp	2009-04-18 12:40:28.000000000 +0300
+++ mysql-connector-c++-1.0.5/driver/mysql_art_resultset.cpp	2010-09-01 21:59:29.000000000 +0300
@@ -17,6 +17,7 @@
 #include <cppconn/exception.h>
 #include <sstream>
 #include <math.h>
+#include <cstdio>
 
 
 namespace sql
diff -N -r -U 3 mysql-connector-c++-1.0.5-orig/driver/mysql_resultbind.cpp mysql-connector-c++-1.0.5/driver/mysql_resultbind.cpp
--- mysql-connector-c++-1.0.5-orig/driver/mysql_resultbind.cpp	2009-04-18 12:40:28.000000000 +0300
+++ mysql-connector-c++-1.0.5/driver/mysql_resultbind.cpp	2010-09-01 22:01:25.000000000 +0300
@@ -13,7 +13,8 @@
 #include "mysql_debug.h"
 #include "mysql_resultbind.h"
 
-#include <string.h>
+#include <cstring>
+#include <cstdio>
 
 namespace sql
 {
diff -N -r -U 3 mysql-connector-c++-1.0.5-orig/test/unit/classes/resultsetmetadata.cpp mysql-connector-c++-1.0.5/test/unit/classes/resultsetmetadata.cpp
--- mysql-connector-c++-1.0.5-orig/test/unit/classes/resultsetmetadata.cpp	2009-04-18 12:40:28.000000000 +0300
+++ mysql-connector-c++-1.0.5/test/unit/classes/resultsetmetadata.cpp	2010-09-01 22:05:20.000000000 +0300
@@ -15,6 +15,7 @@
 #include "resultsetmetadata.h"
 #include <sstream>
 #include <stdlib.h>
+#include <cstdio>
 #include <cppconn/resultset.h>
 #include <cppconn/datatype.h>
 #include <cppconn/prepared_statement.h>
